algorithms/CSharp/README.md

```diff
@@ -7,6 +7,7 @@ For running the `.cs` file please using [.Net Finddle](https://dotnetfiddle.net/
 2. [Insertion Sort](src/Sorts/insertion-sort.cs)
 3. [Selection Sort](src/Sorts/selection-sort.cs)
 4. [Counting Sort](src/Sorts/counting-sort.cs)
+5. [Merge Sort](src/Sorts/merge-sort.cs)
```

Comment:
@ming-tsai by reference, didn't you mean this? I can see it's already in the PR. Let me know if I got it wrong

Reply:
In this case is adding to documentations folder, due to this is algorithm is documented

Reply:
@ming-tsai got it. Adding it asap

Reply:
@ming-tsai done